src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import './App.css'
+import { Global } from '@emotion/react';
+import { globalStyle } from './GlobalStyle';
+import Header from './components/Header';
+import Main from './components/pages/Main';
+import FreeBoard from './components/pages/FreeBoard';
+import Items from './components/pages/Items';
+import { Route, Routes } from "react-router";
+import AddItem from './components/pages/AddItem';
+
+function App() {
+
+  return (
+    <>
+      <Global styles={globalStyle} />
+        <Header />
+    
+        <Routes>
+          <Route path="/" element={<Main />} />
```

Comment:
💊 제안
index 속성을 이용해 시작페이지라는 것을 더 명확하게 알 수 있게 하시는 것을 추천드립니다~

```suggestion
          <Route index element={<Main />} />
```

https://reactrouter.com/start/declarative/routing